ecl/hql/CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ if (WIN32)
     )
 else()
     add_custom_command ( OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.cpp ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.h
+        #delete output files before building to force rebuilds with errors to fail rather than continue unnoticed, using the existing output files.
+        COMMAND ${bisoncmdprefix} rm -f ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.h ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.cpp
```

Comment:
Yep, I should have looked at the actual file rather than just the single line returned the file search for "bisoncmdprefix". I did wonder what "call" even was, I know my cmake knowledge has holes in it and lazily assumed it came from such a hole, I should have inquired.